specification/semantic_conventions.md

```diff
@@ -101,29 +101,35 @@ For each `InstrumentationLibraryLogs` instance:
 
 The resource field in `ResourceLogs` MUST contain the [`telemetry.sdk.language`](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/resource/semantic_conventions#telemetry-sdk) attribute.
 
-### `LogRecord` Messsage Common Attributes
+### `LogRecord` Message Common Attributes
 
 Inside each `InstrumentationLibraryLogs` instance is a list of `LogRecord`
 instances. For each `LogRecord` instance:
 
 - `com.splunk.sourcetype` MUST be set to the value `otel.profiling`
+- `profiling.data.type` MUST be set to either `allocation` or `cpu`
+- `profiling.data.format` MUST be set to either `text` or `pprof-gzip-base64`
+
+#### `LogRecord` Message `text` Data Format Specific Attributes
+
 - `source.event.period` MUST contain the sampling period if this `LogRecord` represents a periodic event
 - `source.event.name` OPTIONALLY can contain the name of the event that triggered the sampling
 - `memory.allocated` MUST contain the allocation size if this `LogRecord` represents a memory allocation event
-- `profiling.data.type` MUST be set to either `allocation` or `cpu`
-- `profiling.data.format` MUST be set to either `text` or `pprof-gzip-base64`
 - `thread.stack.truncated` MUST be set to boolean `true` when this `LogRecord` does not contain the full stack trace
 
 ### `LogRecord` Message Fields
 
 - [Time](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/data-model.md#field-timestamp)
   MUST be set to the time that the call stack was sampled.
```

Comment:
I guess this should also move to the `text` payload?

Reply:
Fixed: dbf4517ce897dd50687ae15eb50ebcad49ef6d12